src/__test__/test-utils/testComponentFactory.jsx

```diff
@@ -6,8 +6,8 @@ const createTestComponentFactory = (Component, defaultProps) => (customProps = {
   // return a new object without mutating defaultProps
   const props = _.merge(
     {},
-    defaultProps,
-    customProps,
+    _.cloneDeep(defaultProps),
+    _.cloneDeep(customProps),
```

Comment:
You don't need `_.cloneDeep` for this. The remaining parameters in `merge` is not modified by `merge`.